spec/helpers/application_theming_helper_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationThemingHelper, type: :helper do
+  describe '#application_resources' do
+    subject { helper.application_resources }
+
+    it { should be_html_safe }
+
+    it 'has application.css' do
+      expect(subject).to have_tag('link', with: {
+                                   href: '/assets/application.css'
+                                 })
+    end
+
+    it 'has application.js' do
+      expect(subject).to have_tag('script', with: {
+                                              src: '/assets/application.js'
+                                            })
+    end
+
+    it 'has jquery' do
+      expect(subject).to have_tag('script', with: {
+                                              src: '/assets/jquery.js'
+                                            })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.